app/services/doc_auth_router.rb

```diff
@@ -56,56 +56,153 @@ def translate_generic_errors!(response)
     # rubocop:enable Style/GuardClause
   end
 
-  def self.client
-    case doc_auth_vendor
-    when 'acuant'
-      AcuantErrorTranslatorProxy.new(
-        IdentityDocAuth::Acuant::AcuantClient.new(
-          assure_id_password: Figaro.env.acuant_assure_id_password,
-          assure_id_subscription_id: Figaro.env.acuant_assure_id_subscription_id,
-          assure_id_url: Figaro.env.acuant_assure_id_url,
-          assure_id_username: Figaro.env.acuant_assure_id_username,
-          facial_match_url: Figaro.env.acuant_facial_match_url,
-          passlive_url: Figaro.env.acuant_passlive_url,
-          timeout: Figaro.env.acuant_timeout,
-          exception_notifier: method(:notify_exception),
-        ),
-      )
-    when 'lexisnexis'
+  class LexisNexisTranslatorProxy
+    attr_reader :client
+
+    def initialize(client)
+      @client = client
+    end
+
+    def method_missing(name, *args, &block)
+      if @client.respond_to?(name)
+        translate_form_response!(@client.send(name, *args, &block))
```

Comment:
not sure if this will be ever a problem on the security context, but can this be changed to use `public_send` instead of `send`?

Reply:
Good question! It might be able to use `public_send`